### PR TITLE
Upgrade package apr to 1.7.2 to fix CVE-2022-24963

### PR DESCRIPTION
**Pull request by suresh-thelkar**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?
Upgrade package apr to 1.7.2 to fix CVE-2022-24963

###### Change Log  

- NA

###### Does this affect the toolchain?  

**NO**

###### Associated issues  

- NA

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2022-24963

###### Test Methodology

Buddy Build
- [AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=308505&view=results)
- [ARM64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=308484&view=results)

### CODE REVIEW

**Review by mfrw — APPROVED** [review]
LGTM thanks

**Comment by suresh-thelkar** [comment]
Thanks @mfrw for the review and approval.

**Review by nicogbg — APPROVED** [review]

File: SPECS/apr/apr.spec [hunk 1]
```diff
@@ -54,41 +56,57 @@ make -j1 check
...
-%files  devel
+%files devel
 %defattr(-,root,root)
 %{_includedir}/*
 %{_libdir}/*.a
 %{_libdir}/*.so
 %{_libdir}/pkgconfig
 
 %changelog
+* Tue Feb 14 2022 Suresh Thelkar <sthelkar@microsoft.com> - 1.7.2-1
```
Comment (jslobodzian) [inline SPECS/apr/apr.spec:67]: Date is wrong.  It's 2023 now.

Comment (suresh-thelkar) [reply SPECS/apr/apr.spec]: Thanks @jslobodzian. I have addressed it. Please look at the latest commit.

**Review by jslobodzian — APPROVED** [review]

**Comment by suresh-thelkar** [comment]
Thank you @jslobodzian / @nicogbg for your reviews and approvals.

**Pull request merged**